prdoc/schema_user.json

```diff
@@ -0,0 +1,223 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema#",
+    "$id": "https://raw.githubusercontent.com/paritytech/prdoc/master/prdoc_schema_user.json",
+    "version": {
+      "major": 1,
+      "minor": 0,
+      "patch": 0,
+      "timestamp": 20230817152351
+    },
+    "title": "Polkadot SDK PRDoc Schema",
+    "description": "JSON Schema definition for the Polkadot SDK PR documentation",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+      "title": {
+        "type": "string",
+        "description": "Title for the PR. This is what will show up in the documentation"
+      },
+
+      "authors": {
+        "description": "Optional: You may provide your name, github handle, or email. That can help finding your 'your' prdoc files later on",
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+
+      "tags": {
+        "description": "Optional: You may provide some tags here. This is currently not used downstream but can help you find some prdoc files later on",
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+
+      "doc": {
+        "type": "array",
+        "description": "Description of the PR. Provide a description for each relevant audience.\nSee the `audience` property for more documentation about audiences",
+        "items": {
+          "$ref": "#/$defs/doc"
+        },
+        "minItems": 1
+      },
+      "crates": {
+        "description": "You have the option to provide input about the semver level of your changes for each crate\n. This is used during the crate publishing to crates.io and to help users understand the impact of the changes introduced in your PR.\nYou can also set it to `no change` if you know there was no significant semver change",
```

Comment:
If you change something in  a crate, what is a "no change" ?

Reply:
Say you work on the rustdoc of the crate or you do refactoring that does ont affect the interfaces, this will be neither a semver change considered as `major`, `minor` or `patch` (we may consider this as `patch` though, more on that below).

I am not a big fan of the `no change` term to be honest, maybe you have a better idea ?
Alternatively, we could also dump this 4th choice and stick with `major`, `minor` or `patch` and default to `patch`.

Reply:
The docs are a good example. However, releasing them also eagerly is probably a good idea. I mean you could also just omit a crate there to have it released.

Reply:
The main consumer of this data is @Morganamilo with `parity-publish `, I am fine with whatever option works best with the downstream tooling and keeps it simple.

@Morganamilo what do you think ?